File: Java/Panda/Factorial.java
import java.util.*;
public class Factorial
{
    public static void main(String args[])
    {
        int a,fact=1;
        int n;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a number");
        n=sc.nextInt();
        for(int i=1;i<=n;i++)
        {
            fact=fact*i;
        }
        System.out.println("Factorial of " + n + " is " + fact);
    }
}

File: C/Factorial.cpp
#include <stdio.h>
int fact(int a)
{
    if(a==0)
    	return 1;
    return a*fact(a-1);
}
int main()
{
	int num;
	printf("Enter a number:\n");
	scanf("%d",&num);
    printf("Factorial of %d is: %d",num,fact(num));
    return 0;
}
